examples/hivemq-swarm/configmap_monitoring.yaml

```diff
@@ -0,0 +1,2372 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: swarm-hivemq-swarm-dashboard
+  namespace: monitoring # Namespace where the prometheus-community/kube-prometheus-stack is installed
+  labels:
+    app.kubernetes.io/instance: swarm # Helm release-name of HiveMQ Swarm chart
+    app.kubernetes.io/name: hivemq-swarm
+    app.kubernetes.io/version: 4.32.0
```

Comment:
do we need this version information here? This outdates quickly

Reply:
These extra labels are not needed. Will remove them. Only the grafana_dashboard is needed.